youtube_dl_gui/locale/bg_BG/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,593 @@
+msgstr "Френски"
+
+#: youtube_dl_gui/optionsframe.py:545
+msgid "German"
+msgstr "Немски"
+
+#: youtube_dl_gui/optionsframe.py:546
+msgid "Greek"
+msgstr "Гръцки"
+
+#: youtube_dl_gui/optionsframe.py:547
+msgid "Hebrew"
+msgstr "Иврит"
+
+#: youtube_dl_gui/optionsframe.py:548
+msgid "Italian"
+msgstr "Италиански"
+
+#: youtube_dl_gui/optionsframe.py:549
+msgid "Portuguese"
+msgstr "Португалски"
+
+#: youtube_dl_gui/optionsframe.py:550
+msgid "Russian"
+msgstr "Руски"
+
+#: youtube_dl_gui/optionsframe.py:551
+msgid "Spanish"
+msgstr "Испански"
+
+#: youtube_dl_gui/optionsframe.py:552
+msgid "Swedish"
+msgstr "Шведски"
+
+#: youtube_dl_gui/optionsframe.py:553
+msgid "Turkish"
+msgstr "Турски"
+
+#: youtube_dl_gui/optionsframe.py:569
+msgid "None"
+msgstr "Без"
+
+#: youtube_dl_gui/optionsframe.py:570
+msgid "Automatic subtitles (YOUTUBE ONLY)"
+msgstr "Автоматични субтитри (САМО ЗА YOUTUBE)"
+
+#: youtube_dl_gui/optionsframe.py:571
+msgid "All available subtitles"
+msgstr "Всички налични субтитри"
+
+#: youtube_dl_gui/optionsframe.py:572
+msgid "Subtitles by language"
+msgstr "Субтитри според езика"
+
+#: youtube_dl_gui/optionsframe.py:578
+msgid "Subtitles"
+msgstr "Субтитри:"
+
+#: youtube_dl_gui/optionsframe.py:582
+msgid "Subtitles options"
+msgstr "Опции на субтитрите:"
+
+#: youtube_dl_gui/optionsframe.py:583
+msgid "Embed subtitles into video file (mp4 ONLY)"
+msgstr "Вграждане на субтитрите във видео файла (САМО за mp4)"
+
+#: youtube_dl_gui/optionsframe.py:585
+msgid "Playlist"
+msgstr "Плейлист"
+
+#: youtube_dl_gui/optionsframe.py:591 youtube_dl_gui/optionsframe.py:596
+msgid "Max"
+msgstr "Макс"
+
+#: youtube_dl_gui/optionsframe.py:594
+msgid "Filesize"
+msgstr "Размер на файл"
+
+#: youtube_dl_gui/optionsframe.py:599
+msgid "Min"
+msgstr "Мин"
+
+#: youtube_dl_gui/optionsframe.py:728
+msgid "Retries"
+msgstr "Повторения"
+
+#: youtube_dl_gui/optionsframe.py:731
+msgid "Authentication"
+msgstr "Удостоверяване:"
+
+#: youtube_dl_gui/optionsframe.py:733
+msgid "Username"
+msgstr "Име"
+
+#: youtube_dl_gui/optionsframe.py:735
+msgid "Password"
+msgstr "Парола"
+
+#: youtube_dl_gui/optionsframe.py:737
+msgid "Video password"
+msgstr "Видео парола"
+
+#: youtube_dl_gui/optionsframe.py:740
+msgid "Network"
+msgstr "Мрежа:"
+
+#: youtube_dl_gui/optionsframe.py:742
+msgid "Proxy"
+msgstr "Прокси"
+
+#: youtube_dl_gui/optionsframe.py:744
+msgid "User agent"
+msgstr "Потребителски агент"
+
+#: youtube_dl_gui/optionsframe.py:746
+msgid "Referer"
+msgstr "Препратка"
+
+#: youtube_dl_gui/optionsframe.py:749
+msgid "Logging"
+msgstr "Отчет:"
+
+#: youtube_dl_gui/optionsframe.py:751
+msgid "Enable log"
+msgstr "Включване на отчет"
+
+#: youtube_dl_gui/optionsframe.py:752
+msgid "View"
+msgstr "Преглед"
+
+#: youtube_dl_gui/optionsframe.py:753
+msgid "Clear"
+msgstr "Изчистване"
+
+#: youtube_dl_gui/optionsframe.py:863
+msgid "Youtube-dl command line options (e.g. --help)"
+msgstr "Опции за коменден ред на Youtube-dl (например --help):"
```

Comment:
Try to be consistent. Here you write **Y**outube-dl (capital Y) but in other lines (165, 169, etc) you write **y**outube-dl. Please pick one and stick with it, unless there's a specific reason for this.